File: betterrandom/src/test/java/io/github/pr0methean/betterrandom/prng/adapter/ReseedingSplittableRandomAdapterDemoTest.java
package io.github.pr0methean.betterrandom.prng.adapter;

import io.github.pr0methean.betterrandom.seed.SeedException;
import org.testng.annotations.Test;

public class ReseedingSplittableRandomAdapterDemoTest {

  private static final String[] NO_ARGS = {};

  @Test(timeOut = 120_000) public void ensureNoDemoCrash()
      throws SeedException, InterruptedException {
    ReseedingSplittableRandomAdapterDemo.main(NO_ARGS);
  }
}

File: betterrandom/src/test/java/io/github/pr0methean/betterrandom/prng/adapter/ReseedingSplittableRandomAdapterDemoTest.kt
package io.github.pr0methean.betterrandom.prng.adapter

import io.github.pr0methean.betterrandom.seed.SeedException
import org.testng.annotations.Test

class ReseedingSplittableRandomAdapterDemoTest {

    @Test(timeOut = 120000)
    @Throws(SeedException::class, InterruptedException::class)
    fun ensureNoDemoCrash() {
        ReseedingSplittableRandomAdapterDemo.main(NO_ARGS)
    }

    companion object {

        private val NO_ARGS = arrayOf<String>()
    }
}
